src/edge_op.cpp

```diff
@@ -317,36 +320,35 @@ void Manifold::Impl::SimplifyTopology() {
     numFlagged = 0;
     ShortEdge se{halfedge_, vertPos_, epsilon_};
```

Comment:
Should this `ShortEdge` check limit to newly added edges? Adding the check to this seems fine.

Reply:
I tried, but it fails when two retained verts are at the same position in space - that edge can't be collapsed, even though it's zero length.